packages/core-database-postgres/lib/connection.js

```diff
@@ -589,6 +589,52 @@ module.exports = class PostgresConnection extends ConnectionInterface {
     return this.cache
   }
 
+  /* Transaction pool aux methods @{ */
+
+  /**
+   * Add transactions to the transaction pool.
+   * @param {Array of Transaction objects} transactions
+   * @return {void}
+   */
+  async transactionPoolAdd (transactions) {
```

Comment:
This should not be part of `core-database-postgres` because you are hardcoding transaction pool logic into a package that is supposed to be decoupled from it.

Reply:
Thanks for looking into this. Indeed it looks a bit strange to have it there. Where else would you suggest to put it?

Btw, assuming we want to store the data in a table in postgres, some functions must be added to core-database-postgres to fiddle with that table?

Reply:
All PostgreSQL traces obliterated in a59ac681.